### DISCUSSION TITLE
train_agentfabric_llm_tool error

### DISCUSSION THREAD

**Discussion by Havenx707** [modelscope/ms-agent#453] [Q&A]
In the 'modelscope-agent/docs/training_llms/train_agentfabric_llm_tool_use_cn.md' file, '--gradient_accumulation_steps $(expr 1 / $nproc_per_node) ' , 这个设置是否有问题（gradient_accumulation_steps =0），导致出现ZeroDivisionError: integer division or modulo by zero错误

**Comment by slin000111** [comment]
fixed, 参考#460
